### DISCUSSION TITLE
Will this work for my needs?

### DISCUSSION THREAD

**Discussion by Valareos** [ruffle-rs/ruffle#878] [General] [answered]
I have a client who has a flash based web component which will not be easy to convert to html5 due to a few reasons.

There is plans to work on one, once the right programmers are identified and can begin the work, but it will take a while before it becomes operational.

Can I load this on their linux based web server, and play the flash file after chrome and other browsers no longer support it?

If so, if anyone has a good tutorial for setting it up, it will be very helpful. Thank you

**Comment by Herschel** [comment, marked as answer]
It depends on the functionality your web component needs; we are still fairly early in development, so much functionality is not in place yet. Particularly we do not have ActionScript 3 support quite yet, but we're working on it. The end goal of this project is to be compatible with as much Flash content as possible, but it will take some time to get there.

The easiest way to try Ruffle is to add the [polyfill](https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#polyfill) to your page, which will automatically replace Flash embeds on your page with Ruffle players.

**Comment by Valareos** [comment]
Hi, as proof of concept, i got it working. It looks weird, but inspecting the page does indeed say it has ActionScript 3 issues. Commented out the change and will keep an eye on new updates! When it starts working for me it should be a simple matter of uploading the new files. Thanks for the help
